packages/patrol/lib/src/native/native_automator.dart

```diff
@@ -974,6 +974,24 @@ class NativeAutomator {
     );
   }
 
+  /// Set mock location
```

Comment:
Does it work on all `iOS/Android` and also on real device? If not, please leave a docs explaining what works and what is not